src/main/java/org/springframework/integration/kafka/listener/QueueingMessageListenerInvoker.java

```diff
@@ -28,7 +32,9 @@
  *
  * @author Marius Bogoevici
  */
-class QueueingMessageListenerInvoker implements Runnable, Lifecycle {
+class QueueingMessageListenerInvoker implements Runnable {
+
+	private static Logger log = LogManager.getLogger(QueueingMessageListenerInvoker.class);
```

Comment:
This is redundant.
Otherwise everything looks good.
Not sure why we have missed that. Similar `stop` issue we faced with `RedisQueueMessageDrivenEndpoint` some days ago... And thanks to XD also.